microsoft-edge/webview2/Roadmap.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Learn what is coming next for WebView2
+title: Roadmap for Microsoft Edge WebView 2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/19/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Microsoft Edge WebView2 Roadmap
+
+The WebView2 control that allows developers to embed web technologies in their native applications. This document outlines the prospective roadmap for WebView2 through 2020 and into 2021. WebView2 is under active development and the roadmap will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here aren't exhaustive and are subject to change. 
+
+If you have concerns or questions about our Roadmap, please leave us feedback in our [feedback repo](https://github.com/MicrosoftEdge/WebViewFeedback).
+
+The WebView2 team has a few major efforts underway:
+
+1. WebView2 Runtime Installer (Q4 2020)
+2. General Availability 
+    - Win32 C/C++ (Q4 2020)
+    - .NET (Q4 2020)
+    - WinUI 3.0 (Q2 2021)
+3. Fixed Version (Q4 2020)
+4. Cross-Platform (TBD)
+
+## WebView2 Runtime Installer
+
+Starting with Edge 84, WebView2's [Evergreen]() distribution model will allow you to target or chain install the WebView2 Runtime onto your user's machine. A preview of the WebView2 Runtime installer will be available Q3 2020 and GA in Q4 2020.
+
+## General Availability 
+
+### Win32 C/C++
+The Win32 C/C++ SDK will GA shortly after the WebView2 Runtime installer GA's in Q4 2020.
+
+### .NET
+
+The .NET SDK wraps the Win32 C/C++ SDK. The .NET SDK will GA within 2 SDK releases after the Win32 C/C++ GA in Q4 2020.
+
+### WinUI 3.0
+You can bring WebView2 to UWP applications through [Win UI 3.0](https://docs.microsoft.com/en-us/uwp/toolkits/winui3/), currently in alpha. Checkout the [Windows UI Library Roadmap](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md) to keep up to date.
+
+## Fixed Version
+WebView2's [Fixed]() distribution model allows you to package the Microsoft Edge binaries inside your native application. Engineering work is currently under way with a preview set to be ready towards the end of  Q3 2020.
+
+## Cross-Platform (TBD)
+
+We plan to eventually bring WebView2 to MacOS, Linux, XBox and Hololens! 
```

Comment:
Let's not call out xbox, hololens, and 10x at all, and not make any commitment on non-windows platforms. Again, the message is we're listening.